examples of code/Eds_code.rb

```diff
@@ -0,0 +1,68 @@
+class Note
```

Comment:
It's good to look at references for your code, but you probably don't want to commit it with the rest of the project.